README.md

```diff
@@ -1,2 +1,69 @@
-# enhancements
-Enhancement Tracking and Backlog Repo for Istio Releases
+# Enhancement Tracking and Backlog
+Enhancement Tracking and Backlog Repo for Istio Releases.
+
+This repo contains issues and Istio enhancement proposals. These issues are umbrellas for new enhancements to be added to Istio. An enhancement usually takes multiple releases to complete. And an enhancement can be tracked as backlog items before work begins. An enhancement may be filed once there is consensus in at least one Istio working group.
```

Comment:
Curious: does the "enhancement usually takes multiple releases to complete" imply a definitive feature-flag / opt-in until completion?

And what is implied about features that take only a single release cycle?

Reply:
not necessarily, an enhancement will mostly likely finish in multiple releases from code drop + testing + graduation from alpha-beta-stable.  For features take a single release cycle is rare, probably very minor things or updates to an existing feature.  Single release cycle enhancement will be tracked also, when criterias meet.